com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2021.1-to-2021.2.md

```diff
@@ -80,3 +80,5 @@ If you used transparent materials (**Surface Type** set to **Transparent**) with
 From 2021.2, HDRP includes an updated `RendererList` API in the `UnityEngine.Rendering.RendererUtils` namespace. This API has lower overhead than the previous version of the `RendererList` API when it submits the RendererList for drawing. You can use this new version to query if the list of visible objects is empty.
 
 The previous version of the API in the `UnityEngine.Experimental.Rendering` namespace is still available for compatibility purposes but is now deprecated.
+
+When the **Dynamic Render Pass Culling** option is enabled in the HDRP Global Settings, HDRP will use the new API to dynamically skip certain drawing passes based on the type of currently visible objects. For example if no objects with distortion are drawn, the Render Graph passes that draw the distortion effect (and their dependencies - like the color pyramid generation) will be skipped.
```

Comment:
Suggested change. I removed "dynamically" because, as a rule, we avoid adjectives in technical writing and I think we describe its behavior clearly: 

When the **Dynamic Render Pass Culling** property is enabled in the HDRP Global Settings, HDRP uses the new `RendererList` API to skip some drawing passes based on the type of GameObjects that are currently visible. For example, if HDRP doesn't draw any GameObjects with distortion, it skips the Render Graph passes that draw the distortion effect and their dependencies like color pyramid generation.